app/uk/gov/hmrc/personaldetailsvalidation/matching/MatchingConnector.scala

```diff
@@ -54,8 +57,11 @@ class MatchingConnectorImpl @Inject()(httpClient: HttpClient, connectorConfig: M
         )
       } recover {
         case ex: UnhealthyServiceException =>
+          platformAnalyticsConnector.sendEvent(GAEvent("sos_iv", "circuit_breaker", "pdv_unavailable_circuit-breaker"), None)
           Left(ex) //GA events (VER-2154& VER-2167) and audit events set up for this circuit breaker (see VER-2152 VER-2153)
-        case ex: Exception => Left(ex)
+        case ex: Exception =>
+          platformAnalyticsConnector.sendEvent(GAEvent("sos_iv", "circuit_breaker", s"${ex.getClass}"), None)
```

Comment:
This event doesn't seem to be required by the ticket description?  It's a general exception, nothing to do with circuit breaker, right?